mono/metadata/object.c

```diff
@@ -7883,12 +7883,14 @@ mono_delegate_ctor_with_method (MonoObjectHandle this_obj, MonoObjectHandle targ
 
 #ifndef DISABLE_REMOTING
 	if (!MONO_HANDLE_IS_NULL (target) && mono_class_is_transparent_proxy (mono_handle_class (target))) {
+#ifdef ENABLE_INTERPRETER
+		if (!method)
```

Comment:
what happens if `method == NULL`?  can this even possible?  i.e. should we put an assert just as in the JIT case?

also this is going to be funky in mixed mode.

Reply:
This !method check shouldn't be here. We don't care about the method, which can't be resolved anyway in interp case, since the addr directly contains the imethod for the target. I need to update.

Yes, delegates will need significant working, regardless of this commit.